scripts/stop.sh

```diff
@@ -1,4 +1,4 @@
-docker-compose -f ./docker/docker-compose.yaml -p veritable-demo down -v
+docker-compose -f ../docker/docker-compose.yaml -p veritable-demo down -v
 sleep 1
 cd ./von-network
 ./manage down
```

Comment:
This is strange.. If I run this from the main root folder I get :

```
DIGI-ABaliou:veritable-poc andy$ ./scripts/stop.sh
ERROR: .FileNotFoundError: [Errno 2] No such file or directory: './../docker/docker-compose.yaml'
\e[33m
Stopping and removing any running containers ...\e[0m
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
DONE
```

If I run this from the scripts folder I get:

```
DIGI-ABaliou:scripts andy$ ./stop.sh
Network von_von is external, skipping
Removing volume veritable-demo_cloudagent-data
WARNING: Volume veritable-demo_cloudagent-data not found.
./stop.sh: line 3: cd: ./von-network: No such file or directory
./stop.sh: line 4: ./manage: No such file or directory
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
DONE
```